src/main/java/no/nav/pensjon/selvbetjeningopptjening/opptjening/MerknadHandler.java

```diff
@@ -163,19 +162,19 @@ private boolean mottattForstegangstjenesteBelop(Beholdning beholdning) {
                 && beholdning.getForstegangstjenesteOpptjeningBelop().getBelop() > 0;
     }
 
-    private void addMerknadOmsorgFromPensjonsbeholdning(int year, List<MerknadCode> merknadList, List<Beholdning> pensjonsbeholdningList) {
+    private void addMerknadOmsorgFromPensjonsbeholdning(int year, OpptjeningDto opptjening, List<MerknadCode> merknadList, List<Beholdning> pensjonsbeholdningList) {
```

Comment:
Stusser litt over at det brukes en DTO i logikken her (burde finnes en domeneklasse `Opptjening`). Men det er kanskje en god grunn for det?